developer-docs-site/docs/concepts/basics-accounts.md

```diff
@@ -5,9 +5,14 @@ id: "basics-accounts"
 
 # Accounts
 
-An account on the Aptos blockchain contains blockchain assets. These assets, for example, coins and NFTs, are by nature scarce and must be access-controlled. Any such asset is represented in the blockchain account as a **resource**. A resource is a Move language primitive that emphasizes access control and scarcity in its representation. However, a resource can also be used to represent other on-chain capabilities, identifying information, and access control. 
+An account on the Aptos blockchain contains blockchain assets. These assets, for example, coins and NFTs, are by nature scarce and must be access-controlled. Any such asset is represented in the blockchain account as a **resource**. A resource is a Move language primitive that emphasizes access control and scarcity in its representation. However, a resource can also be used to represent other on-chain capabilities, identifying information, and access control.
 
-Each account on the Aptos blockchain is identified by a 32-byte account address. An account can store data and it stores this data in resources. The initial resource is the account data itself (authentication key and sequence number). Additional resources like currency or NFTs are added after creating the account. 
+Each account on the Aptos blockchain is identified by a 32-byte account address. An account can store data and it stores this data in resources. The initial resource is the account data itself (authentication key and sequence number). Additional resources like currency or NFTs are added after creating the account.
+
+There are three types of accounts in Aptos:
+  * *Local account* - An account created by you for local testing.
+  * *Blockchain account* - This is the on-chain representation of local account with a different storage location.
+  * *[Resource account](../guides/resource-accounts.md)* - An account used by developers to publish modules and sign transactions.
```

Comment:
Technically a resource account is used for 2 main purposes:
1. Store and isolate resources. This has been the main use case we've seen so far where a module creates a resource account just to host specific resources.
2. Publish module as a standalone (resource) account. This is a building block in a decentralized design where no private keys can control the resource account. The ownership (SignerCap) can be kept in another module, such as governance.

Reply:
Thank you! Added.

Reply:
Can we also clarify here? 

"An account used by developers to publish modules and sign transactions." => "An autonomous account used by developers to store resources or publish modules on chain without a corresponding private key."

Reply:
> Can we also clarify here?
> 
> "An account used by developers to publish modules and sign transactions." => "An autonomous account used by developers to store resources or publish modules on chain without a corresponding private key."

Done!